packages/core/src/shared/settings-amazonq.gen.ts

```diff
@@ -18,7 +18,9 @@ export const amazonqSettings = {
         "amazonQWelcomePage": {},
         "amazonQSessionConfigurationMessage": {},
         "minIdeVersion": {},
-        "ssoCacheError": {}
+        "ssoCacheError": {},
+        "AmazonQLspManifestMessage": {},
+        "AmazonQ-WorkspaceLspManifestMessage":{}
```

Comment:
Why does this not follow the existing naming convention, which is found in the items just above this?